infoTree/requireJSON.js

```diff
@@ -0,0 +1,452 @@
+const URL0 = require("./out/URL0.txt");
```

Comment:
lol, brute force. works for now I guess. I assume 
```
function varMapper(index){ return require("./out/URL" + index + ".txt");
```
doesn't work?

Reply:
nope require is done at compile time :(